zeroconf/__init__.py

```diff
@@ -420,7 +420,7 @@ def entry_to_string(self, hdr: str, other: Optional[Union[bytes, str]]) -> str:
             result += ","
         result += self.name
         if other is not None:
-            result += ",%s]" % cast(Any, other)
+            result += "]=%s" % cast(Any, other)
```

Comment:
Either this belongs in a separate pull request or the name of this one should be changed to "clean up string representations" or something – now it only says it's about using `to_string` in `__repr__` and this is clearly something extra.